src/lib/components/icon/Icon.component.tsx

```diff
@@ -50,26 +52,35 @@ export const iconTable = {
   Sync: 'fas faSync',
   Export: 'fas faFileExport',
   Copy: 'far faClone',
+  'Simple-upload': 'fas faUpload',
   Upload: 'fas faFileUpload',
   'Add-plus': 'fas faPlusSquare',
   'Remove-minus': 'fas faMinusSquare',
   Sort: 'fas faSort',
   'Sort-up': 'fas faSortUp',
   'Sort-down': 'fas faSortDown',
   Calendar: 'fas faCalendarWeek',
+  'Calendar-minus': 'fas faCalendarMinus',
   'Arrow-up': 'fas faArrowUp',
   'Arrow-down': 'fas faArrowDown',
   'Arrow-right': 'fas faArrowRight',
+  'Arrow-left': 'fas faArrowLeft',
+  'Arrow-alt-circle-up': 'fas faArrowAltCircleUp',
   Folder: 'far faFolder',
   File: 'far faFile',
+  'File-invoice': 'fas faFileInvoice',
   'Deletion-marker': 'fas faBan',
+  'Map-marker': 'fas faMapMarkerAlt',
```

Comment:
I don't remember using this one.

Reply:
<img width="2136" alt="Capture d’écran 2022-09-15 à 11 08 43" src="https://user-images.githubusercontent.com/7314118/190364086-242f95c7-b765-4d13-91d6-f153a80334ae.png">

Here

Reply:
I never saw this screen, I think there is always a default storage location on ARTESCA (not on zenko).
Let's keeps this icon.